templates/repo/diff/box.tmpl

```diff
@@ -66,13 +66,15 @@
 				<div class="diff-file-box diff-box file-content {{TabSizeClass $.Editorconfig $file.Name}} mt-3" id="diff-{{$file.NameHash}}" data-old-filename="{{$file.OldName}}" data-new-filename="{{$file.Name}}" {{if $file.ShouldBeHidden}}data-folded="true"{{end}}>
 					<h4 class="diff-file-header sticky-2nd-row ui top attached normal header df ac sb">
 						<div class="df ac">
-							<a role="button" class="fold-file muted mr-2">
-								{{if $file.ShouldBeHidden}}
-									{{svg "octicon-chevron-right" 18}}
-								{{else}}
-									{{svg "octicon-chevron-down" 18}}
-								{{end}}
-							</a>
+						  {{if or (gt $file.Addition 0) (gt $file.Deletion 0)}}
```

Comment:
I can see this being quite error-prone.
The first things that are already broken are non-SVG images such as pngs.
Other media and binary types will probably break just as easily, i.e. audio or video files.
I think it is better to define it as a function that takes the given file as input, and outputs a boolean.

Reply:
Thanks for bringing that to my attention. I haven't thought about that. Fortunately, there is already a variable set to show is a file is a binary file, so this was pretty easy to solve. Everything works now as expected.